app-cryptography/signing-party/autobuild/build

```diff
@@ -0,0 +1,42 @@
+cd "$SRCDIR"
+
+abinfo "Running make to generate files..."
+make
+
+abinfo "Installing files..."
+mkdir -pv "$PKGDIR"/usr/bin
+mkdir -pv "$PKGDIR"/usr/share/man/man1
+
+# To make the following for loop work.
+install -Dvm755 gpgwrap/bin/gpgwrap gpgwrap/gpgwrap
+
+for binname in \
+	caff gpg-key2latex gpg-key2ps gpg-mailkeys gpgdir gpglist \
+	gpgparticipants gpgsigs gpgwrap keyanalyze keyart keylookup \
+	sig2dot springgraph ; do
+	install -Dvm755 "$SRCDIR"/$binname/$binname "$PKGDIR"/usr/bin/$binname
+	find $binname -type f -iname '*.1' -exec cp -v {} "$PKGDIR"/usr/share/man/man1/ \;
+done
+
+abinfo "Installing extra files..."
+for binpath in \
+	caff/pgp-clean caff/pgp-fixkey gpgparticipants/gpgparticipants-filter gpgparticipants/gpgparticipants-prefill \
+	keyanalyze/pgpring/pgpring keyanalyze/process_keys ; do
+	install -Dvm755 $binpath /usr/bin/${binpath/#*\//}
+done
+
+# List obtained from Arch Linux:
+# https://gitlab.archlinux.org/archlinux/packaging/packages/signing-party/-/blob/main/PKGBUILD?ref_type=heads
+abinfo "Installing examples..."
+mkdir -pv "$PKGDIR"/usr/share/doc/$PKGNAME/{caff,gpgsigs,keyanalyze,keyart}
+for example in \
+	caff/caffrc.sample gpgsigs/gpgsigs-lt2k5.txt gpgsigs/gpgsigs-lt2k5-annotated.txt keyanalyze/analyze.sh \
+	keyanalyze/allkeys.sh keyanalyze/scripts keyanalyze/willy keyart/doc ; do
+	cp -rv $example "$PKGDIR"/usr/share/doc/$PKGNAME/$example
+done
+# Remove keyart.1 in examples
+rm -v "$PKGDIR"/usr/share/doc/$PKGNAME/keyart/doc/keyart.1
+
+# Install README and LICENSE files
+abinfo "Installing README and LICENSE files..."
+find -type f -name README -and -name LICENSE -exec install -Dvm644 {} "$PKGDIR"/usr/share/doc/$PKGNAME/{} \;
```

Comment:
Why? Are there multiple ones? The current implementation will overwrite files at every instance.

Reply:
Yes, there are multiple LICENSEs and READMEs

Reply:
> Yes, there are multiple LICENSEs and READMEs

So you'd need to use for loop or at least create sub-directories (but generally, suffixes are better).

Reply:
This line includes the relative path of the found files, both in `find` itself and the target path of `install`, so there should be no problem.